### ISSUE TITLE
iTerm2 cursor color either transparent or same as background

### ISSUE THREAD

**Issue by JohnHuddleston** [dylanaraps/pywal#382]
Every wallpaper I've attempted to use produces the same result: an invisible cursor.  It makes it impossible to work in vim, the only way to figure out where you are is to find a pair of brackets/parentheses/etc. that will highlight.

Running on macOS High Sierra 10.13.6 using pywal as the backend.  I can provide any more details necessary.

**Comment by dylanaraps** [comment]
I'm going to have to wait for a PR for this one as I don't have a macOS machine to test on.

**Comment by zzzeyez** [comment]
i've also noticed this.  i have spent all morning digging through the wal.vim colorscheme to see if i can change the the cursor background

**Comment by alexvilanovab** [comment]
In my case it's not only the vim cursor but the iTerm one.

**Comment by JohnHuddleston** [comment]
I should mention, I used vim to explain how hard the impact of the bug is.  The problem is terminal-based and applies to all terminal-based applications.

**Comment by robwaz** [comment]
@JohnHuddleston This issue should be resolved.  I experienced the same issue and corrected it with the pull request shown above.

**Comment by JohnHuddleston** [comment]
Awesome, confirmed on my end, everything works!

**Comment by phlind** [comment]
When will those changes be available in the pip version? The latest version I can install is from the 21st January.

**Comment by zjp** [comment]
@phlind `pip3 install pywal` on macOS does still pull a version of pywal without the bugfix, but I found that if I copied the current contents of sequences.py from github into /usr/local/lib/python3.7/site-packages/pywal/sequences.py then the issue was resolved without issue.

**Comment by zzzeyez** [comment]
this issue has returned for me

**Comment by BuzzwordChief** [comment]
+1

**Comment by BuzzwordChief** [comment]
Building from the master branch should work, had the same issue with release 3.3.0

**Comment by zzzeyez** [comment]
yeah, i had forgotten about this and reinstalled.  i just made the edit to my pip install
